code/__HELPERS/logging.dm

```diff
@@ -84,4 +84,11 @@
 /proc/log_to_dd(text)
 	world.log << text //this comes before the config check because it can't possibly runtime
 	if(config.log_world_output)
-		diary << "\[[time_stamp()]]DD_OUTPUT: [text][log_end]"
\ No newline at end of file
+		diary << "\[[time_stamp()]]DD_OUTPUT: [text][log_end]"
+
+/**
+ * Standardized method for tracking startup times.
+ */
+/proc/log_startup_progress(var/message)
+	world << "<span class='danger'>[message]</span>"
+	world.log << message
```

Comment:
This should probably use log_to_dd, so that the world start messages get sent to the server logs as well.